src/hooks/tradingView/useTradingViewDatafeed.ts

```diff
@@ -104,6 +107,12 @@ export const useTradingViewDatafeed = ({ tickSizeDecimals, orderbookCandlesToggl
 
         resolveSymbol: async (symbolName: string, onSymbolResolvedCallback: ResolveCallback) => {
           const symbolItem = getSymbol(symbolName || DEFAULT_MARKETID);
+
+          const { tickSizeDecimals: tickSizeDecimalsAbacus } = orEmptyObj(
+            getCurrentMarketConfig(store.getState())
```

Comment:
should we be looking for the symbolName? I am slightly worried about things being slightly out of sync or something.

Reply:
donezo